dmn-evaluation-provider-api/pom.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>dmn</artifactId>
+        <groupId>org.powerflows</groupId>
+        <version>2.0.0-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>dmn-evaluation-provider-api</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.powerflows</groupId>
+            <artifactId>dmn-model</artifactId>
+            <version>${parent.version}</version>
+            <scope>provided</scope>
```

Comment:
why provided? classes from this dependency are used in code of this module, it should be required dependency

Reply:
changed